builtin/logical/pki/storage.go

```diff
@@ -1394,6 +1394,12 @@ func (sc *storageContext) getAutoTidyConfig() (*tidyConfig, error) {
 		result.IssuerSafetyBuffer = defaultTidyConfig.IssuerSafetyBuffer
 	}
 
+	// If revoked safety buffer is not specified OR it points to the default of safety buffer while safety buffer is specified
+	if result.RevokedSafetyBuffer == nil || (*result.RevokedSafetyBuffer == defaultTidyConfig.SafetyBuffer && result.SafetyBuffer != defaultTidyConfig.SafetyBuffer) {
```

Comment:
I don't think the latter part of this conditional will ever hold.

Reply:
I'd also do this same fix on line 1385 to "fix" the returned default tidy config value (set `result.RevokedSafetyBuffer = &revokedBufferValue`) -- not modifying the default, just the returned one of course.

Reply:
The condition `result.SafetyBuffer != defaultTidyConfig.SafetyBuffer` was used because I was defaulting revoked to the default cause of safety buffer when it was not specified, the logic has been updated and the redundant check has also been removed :)